src/lib/arch/riscv/InstructionMetadata.cc

```diff
@@ -252,6 +254,166 @@ void InstructionMetadata::alterPseudoInstructions(const cs_insn& insn) {
       }
       break;
     }
+
+    case Opcode::RISCV_CSRRS: {
+      if (operandCount == 1 && strcmp(mnemonic, "frflags") == 0) {
+        // frflags Rs is pseudo of CSRRS Rs, fflags, zero (Read FP exception
+        // flags) CSRRS Rs, _, _ -> CSRRS Rs, fflags, zero
+        operands[1].type =
+            RISCV_OP_IMM;  // TODO needs to become reg when CS updated
+        operands[1].reg = RISCV_SYSREG_FFLAGS;  // fflags address
+
+        operands[2].type = RISCV_OP_REG;
+        operands[2].reg = 1;
+
+        operandCount = 3;
+      } else if (strcmp(mnemonic, "rdinstret") == 0) {
+        assert(false && "Unimplemented psuedoinstruction rdinstret");
```

Comment:
All statements which print to the stdout or stderr need to be prefixed. Please add `[SimEng::InstructionMetadata]` to the beginning of all assert strings in this file

Reply:
Sorry, this mistake is on me... the prefix should actually be `[SimEng:InstructionMetadata]`